ghcide/src/Development/IDE/Plugin/TypeLenses.hs

```diff
@@ -246,8 +244,12 @@ rules = do
     result <- liftIO $ gblBindingType (hscEnv <$> hsc) (tmrTypechecked <$> tmr)
     pure ([], result)
 
-parseCustomConfig :: A.Object -> Maybe Mode
-parseCustomConfig = A.parseMaybe (A..: "mode")
+readMode :: T.Text -> Mode
+readMode = \case
+  "always"      -> Always
+  "exported"    -> Exported
+  "diagnostics" -> Diagnostics
+  _             -> error "failed to parse type lenses mode"
```

Comment:
I think this should return `Always` like the old behavior. People can still modify the configs by hand and ignore the schema.

Reply:
The last branch can never be reached, since `usePropertyLsp` will return `Always` if parse error occurs

Reply:
Please add a comment to that effect!